### PR TITLE
chore(build kong) get wrpc protos from spec

### PR DESCRIPTION
**Pull request by Suika-Kong**
Fix FT-3175

### CODE REVIEW

File: build-kong.sh [hunk 1]
```diff
@@ -44,9 +44,19 @@ pushd /kong
...
+  # clone dp-spec
+  DP_SPEC = /tmp/kong-dp-spec
+  git clone https://github.com/Kong/kong-dp-spec ${DP_SPEC} --recursive
+  git -C ${DP_SPEC} checkout ${KONG_DP_SPEC_VERSION:-f9432f8}
   # /usr/local/kong/include is usually created by other C libraries, like openssl
   # call mkdir here to make sure it's created
   mkdir -p /tmp/build/usr/local/kong/include
+  # wrpc related protos
+  cp -r ${DP_SPEC}/spec/proto/* /tmp/build/usr/local/kong/include/
+  mkdir /tmp/build/usr/local/kong/include/wrpc
```
Comment (chronolaw) [inline build-kong.sh:57]: use `mkdir -p`?

Comment (Suika-Kong) [reply build-kong.sh:57]: We are sure the parent directory exists, and wrpc does not

**Comment by mayocream** [comment]
Friendly reminder that https://github.com/Kong/kong-dp-spec is a private repo.

**Comment by Suika-Kong** [comment]
> Friendly reminder that https://github.com/Kong/kong-dp-spec is a private repo.

How could I forget that... closing this PR

**Comment by fffonion** [comment]
let's hold merge for this one and discuss potential caveats in the ticket

**Pull request closed without merging**